Interoperability/python/src/qsharp/clients/iqsharp.py

```diff
@@ -110,7 +113,22 @@ def estimate(self, op, **params) -> Dict[str, int]:
             for operation_name, count in raw_counts.items()
         }
 
-    def execute(self, input, return_full_result=False, raise_on_stderr=False, **kwargs):
+    def component_versions(self) -> Dict[str, LooseVersion]:
```

Comment:
```suggestion
    def component_versions(self, **kwargs) -> Dict[str, LooseVersion]:
```
so we can pass extra parameters.